app/grants/serializers.py

```diff
@@ -138,11 +139,13 @@ class Meta:
         fields = ('grant_name', 'asset', 'timestamp', 'grant_amount', 'gitcoin_maintenance_amount', 'grant_usd_value', 'gitcoin_usd_value')
 
 
-class GrantCLRSerializer(serializers.ModelSerializer):
+class GrantCLRSerializer(FlexFieldsModelSerializer):
     """Handle metadata of CLR rounds"""
     class Meta:
         """Define the GrantCLR serializer metadata."""
         model = GrantCLR
-        fields = (
-            'id', 'display_text', 'round_num', 'is_active', 'start_date', 'end_date'
-        )
+        fields = '__all__'
```

Comment:
- would this break the public API users are using right now?
- we don't want to expose all info, I'd suggest creating a new serializer with only needed fields and not changing this 
The fields you'd need are 
```
'id', 'total_pot', 'start_date', 'end_date',
        'display_text', 'round_num', 'customer_name', 'sub_round_slug',
        'grant_filters', 'collection_filters' 
```